src/main/java/com/plivo/api/models/message/MessageLister.java

```diff
@@ -63,6 +72,29 @@ public MessageLister errorCode(final Long errorCode) {
     return this;
   }
 
+  @Override  
+  /**
+  * Actually list instances of the resource.
+  */
+  public ListResponse<Message> list() throws IOException, PlivoRestException {
+    validate();
+    Response<ListResponse<Message>> response = obtainCall().execute();
+
+    handleResponse(response);
+    
+    try 
+    {
+      Field meta = response.body().getClass().getDeclaredField("meta");
```

Comment:
If handleResponse throws the exceptions, is this will work?

Reply:
If handleResponse throws exceptions, then behaviour will be same as previous right? the request will throw an exception without returning anything?